src/main/java/emissary/core/IBaseDataObjectHelper.java

```diff
@@ -215,4 +218,40 @@ public static void addParentInformationToChildren(final IBaseDataObject parent,
             }
         }
     }
+
+    /**
+     * Search for the first preferred view by regular expression or use the primary data if none match
+     *
+     * @param payload the payload to pull data from
+     */
+    public static byte[] findPreferredDataByRegex(final IBaseDataObject payload, List<Pattern> preferredViewNamePatterns) {
```

Comment:
Why prefix this method name with "find" but prefix the non-regex one with "get"?  Aside from the pattern usage, they're provide very parallel functionality, with both falling back to using `data()` is a suitable view isn't present.

Reply:
The "get" method name was a direct copy from the `Analyze` class. I'll update the method name to be consistent.